src/components/ActiveClients.js

```diff
@@ -132,9 +132,22 @@ const ActiveClients = () => {
               const { key, debugId, displayDate, os, appName, geo } = debugTag;
               return (
                 <tr key={key}>
+                  <td>{debugId}</td>
                   <td>
-                    <Link className='text-decoration-none' to={`/pings/${debugId}`} onClick={() => {recordClick('Debug ID')}}>
-                      {debugId}
+                    <Link
+                      className='text-decoration-none'
+                      to={`/pings/${debugId}`}
+                      onClick={() => {
+                        recordClick("Debug ID Pings");
+                      }}
+                    >
```

Comment:
Nothing to do here, formatters being formatters...  but I have "Opinions" about breaking up a markup tag like this and what it does to the flow of reading HTML for me.

Reply:
I agree with this. The way that React passes props to components makes every component end up looking like this. Definitely not a pretty way to do it.

Though I will say if you look at React code enough you'll get so used to this formatting that seeing everything on one line will start to look wrong haha.